react/features/screenshot-capture/ScreenshotCaptureSummary.js

```diff
@@ -1,36 +1,40 @@
 // @flow
 
-import pixelmatch from 'pixelmatch';
+import resemble from 'resemblejs';
```

Comment:
Why was this dependency changed? I tried to figure it out from the commit messages but they don't have any meaningful information. Please be **very thorough** with your comments. People should not need to ask you why a change was made, the extended commit message should contain that information.

Reply:
Sorry about that, I started explaining in the commits, but this was from a while ago.
The dependency was changed because the new one can return early when comparing images. The old one compared the whole image and returned the difference, the new returns early if it reaches the given percent difference

Reply:
Sounds good, make sure that is part of the commit description please.